src/test/java/bt/TestUtils.java

```diff
@@ -0,0 +1,22 @@
+package bt;
+
+import bt.sample.Hello;
+import signumj.entity.SignumValue;
+import signumj.entity.response.AT;
+
+import static org.junit.Assert.assertNotNull;
+
+public class TestUtils extends BT {
+
+	public static <T extends Contract> AT registerContract(Class<T> contractClass) {
```

Comment:
Why not add this directly to the BT class and adjust there directly?

Reply:
Is the BT class only used in tests ? if yes ok. If no, then it should be separated.